functions/_fzf_configure_bindings_help.fish

```diff
@@ -12,6 +12,7 @@ DESCRIPTION
         Search git status  |  Ctrl+Alt+S (S for status)    |  --git_status
         Search history     |  Ctrl+R     (R for reverse)   |  --history
         Search variables   |  Ctrl+V     (V for variable)  |  --variables
+        Search processes   |  Ctrl+O     (O for prOcess)   |  --pids
```

Comment:
this says 0 but you got it bought to o. which one do you think is best?

Reply:
Might it be a font issue? I've verified again all letters are O (capital o) and not Zero
The binding is done to \co to mimic the way Ctrl-R is bound to \cr (and since Ctrl binding are case insensitive).
Therefore, Ctrl-O is bound to the Process list.

Am I missing something?

Reply:
Oops my bad it is a capital o indeed. Ok I think that's fine...
Btw can you make the option --processes instead of --pids?